app/views/posts/new/_form.html.erb

```diff
@@ -1,12 +1,16 @@
-<%= form_with model: @post, url: posts_path, method: 'post' do |form| %>
+<%= form_with model: post, url: posts_path, method: 'post' do |form| %>
+
+  <% if !post.parent_id.nil? %> 
```

Comment:
in my way I recommended user `blank?`, but what do you think about it?

Reply:
good, it seems to ensure its presence even more